src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -265,14 +261,16 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
     ]
   );
 
+  const backgroundImageClass = `VenueRoomsEditor__background-image`;
```

Comment:
```suggestion
```

Reply:
fixed in 8fdf201a84ce0a3c21cf962f6ddb483cce0a2c54